examples/offline_inference_openai.md

```diff
@@ -0,0 +1,172 @@
+# Offline Inference with the OpenAI Batch file format
+
+ *NOTE:* This is a guide to performing batch inference using the OpenAI batch file format, *NOT* the complete Batch (REST) API.
+ 
+ ## File Format
+ 
+ The OpenAI batch file format consists of a series of json objects on new lines.
+ 
+ [See here for an example file.](https://github.com/vllm-project/vllm/blob/main/examples/openai_example_batch.jsonl)
```

Comment:
These examples links point to master, so they don't work now, but should work once merged.